test/test_phil.py

```diff
@@ -43,6 +44,6 @@ def test(DataBlockFactory, ExperimentListFactory, flex):
   assert isinstance(params.input.datablock.data, mock.Mock)
   assert isinstance(params.input.experiments.data, mock.Mock)
   # Check we had the correct calls made
-  flex.reflection_table.from_pickle.assert_called_once_with(reflections_path)
+  flex.reflection_table.from_msgpack_file.assert_called_once_with(reflections_path)
```

Comment:
Hi, I don't understand this change.  It appears a pickle test is being removed?

Reply:
@jmp1985 Could you comment here? Is it the case that both lines should be supported in the test?

Reply:
In this test, only the from_msgpack_file method is called. But it looks like pickle files will still be supported in the phil parser.

Reply:
My understanding of the last phone conference with Berkeley is that this is the only obstacle to clear before we can continue tackling [the todo list](https://github.com/dials/dials/pull/535#issuecomment-418088191) on the way to merge. Essentially there is an objection to replacing a pickle test with a message pack test. Instead, a message pack test should be added alongside the existing pickle test, to ensure compatibility with both formats continues to be tested. @phyy-nx @nksauter have I summarised your position accurately?

Reply:
I will add this test

Reply:
I have now modified the test such that it reads in both a msgpack and pickle file